### ISSUE TITLE
Error while using topoplot() function

### ISSUE THREAD

**Issue by k0ruy** [raphaelvallat/yasa#140]
While using topoplot() function, I get an error that mne.viz.plot_topomap() does not have some parameters (`vmin` and `vmax`, instead it only has a `vlim` parameter which takes a tuple of the two limits). I solved this by updating the function parameters as they are on the mne documentation. I suppose the mne library was updated, but the current yasa version uses the old version of men. 
Also when passing the `kwargs` arguments mne says that there is no `show_names` keyword argument.

packages versions:
```
yasa = 0.6.3
numpy =  1.23.5
mne = 1.3.1
scipy = 1.10.1
Python = 3.10.9
```

**Comment by raphaelvallat** [comment]
Hi @k0ruy,

Apologies for the late response. This is a known issue that has been fixed in the development version of YASA (#119). This will be released in the main (stable) version of YASA within the next weeks or months.

Thanks,
Raphael

**Comment by raphaelvallat** [comment]
Just released a new version of YASA that includes the fix: https://github.com/raphaelvallat/yasa/releases/tag/v0.6.4
